docs/services/vision/segmentation.md

```diff
@@ -191,7 +191,7 @@ The following parameters are available for a `"detector_3dsegmenter"`.
 
 | Parameter | Inclusion | Description |
 | --------- | --------- | ----------- |
-| `detector_name`| _Required_  | The name of the detector already registered in the Vision Service that will be turned into a segmenter. |
+| `detector_name`| _Required_  | The name of a registered detector vision service. The segmenter vision service uses the detections from `"detector_name"` to create the 3D segments. |
```

Comment:
I've changed this to something else than you requested on the ticket because your suggestion means the user need to know what the GetDetections API is and I think that's not necessary. LMK what you think @bhaney

Reply:
This sounds good to me!